src/libraries/System.ObjectModel/src/System/ComponentModel/TypeConverterAttribute.cs

```diff
@@ -22,6 +22,8 @@ public sealed class TypeConverterAttribute : Attribute
         /// Initializes a new instance of the <see cref='System.ComponentModel.TypeConverterAttribute'/>
         /// class with the default type converter, which is an empty string ("").
         /// </summary>
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2079:DynamicallyAccessedMembersNotSatisfied",
```

Comment:
We probably didn't update linker in a while. I fixed this a couple weeks ago. The annotation shouldn't be needed: https://github.com/mono/linker/pull/1700

Reply:
We do update linker very ofter and track the tip of linker master most of time

Reply:
In that case this feels like linker bug...

Reply:
It may just have been an outdated suppression in the xml file. I don’t think I checked the warning still occurred in the product before adding the suppression in code. I will verify when I get into the office.

Reply:
Yep - this was just an outdated xml suppression. I've removed this annotation.